### ISSUE TITLE
Your package do not work for more than 1 postc (mediator-outcome confounders affected by the exposure) 

### ISSUE THREAD

**Issue by JuwelRana19** [BS1125/CMAverse#23]
Hi, I tried to reach you several times by email from last December but did not get any reply! I am not sure if you received my email or not. However, I found your package as a comprehensive tool but I am having some problems using the CMAverse package for accounting mediator-outcome confounders affected by the exposure. **First,** it shows the following error if we use more than 1 postc. Is it the limitation of the current package or an error or I made mistake in coding? 

```
res_gformula <- cmest(data = data, model = "gformula", outcome = "Y", exposure = "A",
                  mediator = "M", basec = c("C2"), postc = c("L",  "C1"), EMint = TRUE,
                   mreg = list("logistic"), yreg = "logistic", postcreg = list("linear", "linear"),
                 astar = 0, a = 1, mval = list(1), 
                 estimation = "imputation", inference = "bootstrap", nboot = 2)
                                                                                                                                                                                
 |   0%Error in estinf() : 
  For postcreg[[2]], please regress postc[2] on variables in c(exposure, basec)
```

Second, we think that CMAverse has some typos in some formulas and doesn’t give the same estimates as SAS macro / PROC CAUSALMED. 

It would be useful if you kindly check these issues or let us know if we should avoid using more than 1 postc until it gets fixed. 
@cchoirat @LindaValeri

**Comment by BS1125** [comment]
Hi, thank you for reaching out! Unfortunately, we didn't receive your emails. 
The first issue has been fixed in Commit [90ead13]. Thank you for notifying us. Please update the package and let us know if it works now.
For the second, would you mind specifying the typos? 
Also, when you use the same variables, the same regression models and the regression-based approach (model = "rb") with closed-form parameter function estimation (estimation = "param") and delta method inference (inference = "delta"), you would get the same results with PROC CAUSALMED. Otherwise, the estimates are not expected to be the same.

**Comment by JuwelRana19** [comment]
Thank you so much for fixing the issue. I really appreciate this. Now it works! 
For the second point, I will try your new suggestion and let you know if there are any more issues.
